ansible/pygrid_deploy.yaml

```diff
@@ -0,0 +1,90 @@
+- hosts: hosts
+  remote_user: ubuntu
+  gather_facts: False
+  pre_tasks:
+    - name: Install python
+      raw: bash -c "test -e /usr/bin/python || (sudo apt -qqy update && sudo apt install -qqy python-minimal python2.7)"
+      register: output
+      changed_when: output.stdout
+    - setup: # aka gather_facts
+  
+  tasks:
+    - name: Update apt
+      apt:
+        upgrade: yes
+        update_cache: yes
+      become: true
```

Comment:
I'd rather leave privilege escalations to be configured on `ansible.cfg` than having `become: true` on every task, and then you could specify on `README.md` what should be configured